src/main/java/configuration/ApplicationConfiguration.java

```diff
@@ -0,0 +1,71 @@
+package configuration;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import twitter4j.Twitter;
+import twitter4j.TwitterFactory;
+import twitter4j.conf.ConfigurationBuilder;
+
+import java.io.File;
+
+public class ApplicationConfiguration extends io.dropwizard.Configuration {
+    @JsonProperty
+    private String oauthconsumerkey;
+    @JsonProperty
+    private String oauthconsumersecret;
+    @JsonProperty
+    private String oauthaccesstoken;
+    @JsonProperty
+    private String oauthaccesstokensecret;
+
+    private static Logger logger = LoggerFactory.getLogger(ApplicationConfiguration.class);
+    private String path;
+
+    /** Constructor, sets path to default value
+     * */
+    public ApplicationConfiguration() {
+        this.path = "./config.yml";
+        logger.debug("Configuration created with empty constructor");
+    }
+
+    /** Constructor
+     * @param path: value for Configuration's path property
+     * */
+    public ApplicationConfiguration(String path){
+        this.path = path;
```

Comment:
btw i was thinking that you could use this constructor for your testing config file. that way you won't require all devs to have the `config.yml` for tests to pass that rely on configuration.

Reply:
The issue is that then i will not have 100% coverage because I will not have tested the empty constructor

Reply:
irl you will never see 100% coverage. 70-80 is considered as great coverages for enterprise software. 

not saying you shouldn't attempt to have good coverage. but 100% is an unrealistic goal to have